src/blsCompatibility.spec.ts

```diff
@@ -0,0 +1,1026 @@
+import { FpLegendre, isNegativeLE } from "@noble/curves/abstract/modular";
```

Comment:
Wrote the transaction logic within a test file, to be refactored & moved to production logic when appropriate.